### PR TITLE
Updates for SDL conformance, copyright and alignment changes and Reset the system post SIE SCT run

### PR DESCRIPTION
**Pull request by amrathesh**

### CODE REVIEW

File: bbsr/config/sie_SctStartup.nsh [hunk 1]
```diff
@@ -81,8 +81,11 @@ for %i in 0 1 2 3 4 5 6 7 8 9 A B C D E F
                         if  exist FS%i:\EFI\BOOT\bbr\SCT\Sequence then
                             cp -r FS%i:\EFI\BOOT\bbr\SCT\Sequence\BBSR.seq FS%j:\acs_results\SIE\sct_results\
                         endif
+                        #Restart to avoid an impact of running SCT tests on rest of the suites
+                        echo Reset the system ...
+                        reset
                     endif
-                    goto Done
+                    #goto Done
```
Comment (edhay) [inline bbsr/config/sie_SctStartup.nsh:88]: Why this change?

Comment (amrathesh) [reply bbsr/config/sie_SctStartup.nsh]: Removed the commented line
